styles.css

```diff
@@ -662,4 +846,113 @@
     margin-top: 2rem;
     grid-column: 1/4;
   }
+
+  #contact {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-image: url(../assets/contact2.png);
```

Comment:
- The background image for this section is not displaying, a possible reason for this is that you are navigating into the `assests` directory wrong. Please use `background-image: url(./asseets/contact2.png)`. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your Project:
<img width="207" alt="tydyj" src="https://user-images.githubusercontent.com/73607512/213160549-f1c0065d-cbb6-4fc9-813a-25c51d418a46.PNG">